perun-core/src/test/java/cz/metacentrum/perun/core/impl/DatabaseManagerImplIntegrationTest.java

```diff
@@ -0,0 +1,67 @@
+package cz.metacentrum.perun.core.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import cz.metacentrum.perun.core.AbstractPerunIntegrationTest;
+import cz.metacentrum.perun.core.api.DBVersion;
+import cz.metacentrum.perun.core.api.exceptions.InternalErrorException;
+import cz.metacentrum.perun.core.bl.DatabaseManagerBl;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+/**
+ * @author Simona Kruppova 410315
+ * @author Oliver Mrazik 410035
+ */
+public class DatabaseManagerImplIntegrationTest extends AbstractPerunIntegrationTest{
```

Comment:
These tests just try if parsing of DB version is OK. Do not work with Database itself.